tn_api_v2_core.yml

```diff
@@ -0,0 +1,947 @@
+                brands:
+                    $ref: '#/components/schemas/DatasourceBrandDetails'
+                price_breakdown_with_passenger_type_code:
+                    type: array
+                    description: Displays the pricing for each passenger type
+                    items:
+                        $ref: '#/components/schemas/DatasourcePriceBreakdownDetails'
+                plating_carrier:
+                    type: string
+                    description: Plating carrier. Example = "WS"
+                    example: "WS"
+                intermediate_stops:
+                    type: array
+                    description: Direct flight between two points with no change in flight number
+                    items:
+                        $ref: '#/components/schemas/DatasourceLegDetails'
+                price:
+                    type: number
+                    description: Price of the segment (fare and tax included).
+                    example: 123.45
+                e_ticketability:
+                    type: boolean
+                    description: E-tickable tickets
+                    example: true
+                cancel_penalty:
+                    type: object
+                    description: Information about cancellation
+                    properties:
+                        amount:
+                            type: number
+                            description: Cancellation penalty amount
+                            example: 0
+                        percentage:
+                            type: number
+                            description: Cancellation penalty percentage
+                            example: 100
+                change_penalty:
+                    type: object
+                    description: Information about changing.
+                    properties:
+                        amount:
+                            type: float
+                            description: Change penalty amount
+                            example: 78
+                        percentage:
+                            type: float
+                            description: Change penalty percentage
+                            example: 0
+
+        MinimalDatasourceLegDetails:
+            description: A list of legs for this segment
+            type: object
+            required:
+                - departure_time
+                - arrival_time
+                - flight_number
+                - transportation_type
+            properties:
+                departure_time:
+                    type: string
+                    description: Departure time. Example = "2023-03-15T08:00:00"
+                    example: "2023-03-15T08:00:00"
+                arrival_time:
+                    type: string
+                    description: Arrival time. Example = "2023-03-15T10:29:00"
+                    example: "2023-03-15T10:29:00"
+                flight_number:
```

Comment:
Should include the carrier for the flight separately if we're doing the flight number here. For example if we expect 391 here, we need "AC" for Air Canada as the Carrier.

Reply:
good point, will change